src/AccessibilityInsights.SharedUx/Controls/TestTabs/ColorContrast.xaml.cs

```diff
@@ -119,13 +120,29 @@ private void RunAutoCCA(Bitmap bitmap)
 
             if (pair == null)
             {
+                Logger.PublishTelemetryEvent(
+                    TelemetryAction.ColorContrast_AutoDetect,
+                    new Dictionary<TelemetryProperty, string>
+                    {
+                        [TelemetryProperty.BitmapSize] = (bitmap.Width * bitmap.Height).ToString(CultureInfo.InvariantCulture),
+                        [TelemetryProperty.Confidence] = "None"
+                    }
+                );
                 throw new InvalidOperationException("Unable to determine colors!");
```

Comment:
This also raises a (now redundent) `TelemetryEvent`. How do we suppress the event from this exception?

Reply:
I believe this exception gets caught in line 223 of CCAModeControl.xaml.cs. The canonical way to do this would be to throw a custom exception type, then detect the type and handle it differently. Axe.Windows has an `ExcludedException` for just this purpose, but I would not recommend using that class directly. All things being equal, I'd probably just live with the redundant telemetry.